inline_verifier.go

```diff
@@ -435,6 +436,13 @@ func (v *InlineVerifier) VerifyDuringCutover() (VerificationResult, error) {
 		return VerificationResult{}, err
 	}
 
+	if v.rowsChecked == 0 {
+		return VerificationResult{
+			DataCorrect: false,
+			Message:     "cutover verification failed, no rows were compared",
+		}, nil
+	}
```

Comment:
I wasn't sure how to test this in the ruby integration tests since in integrationferry [here](https://github.com/Shopify/ghostferry/blob/master/test/lib/go/integrationferry.go#L164) we only pass the incorrect tables. Any ideas?